crowdin.yml

```diff
@@ -0,0 +1,12 @@
+project_id: 501094
+api_token: "CROWDIN_PERSONAL_TOKEN"
```

Comment:
This is the way to go, I believe:

```suggestion
```

Reply:
I think this can't help us fix the error. You can check the details [here](https://github.com/zeitgeistpm/documentation/runs/6362825406?check_suite_focus=true)